src/app/datapoints-graph/datapoints-graph-config/datapoints-graph-widget-config.component.html

```diff
@@ -47,6 +41,31 @@
         <label>{{ 'Display options' | translate }}</label>
         <fieldset class="c8y-fieldset m-b-24 m-t-0">
           <c8y-form-group class="p-b-16 m-b-0 p-t-8">
+            <label
+              [title]="'Enable date selection in the widget view.' | translate"
+              class="c8y-checkbox"
+            >
```

Comment:
lack of this checkbox was UX decision by Carlos. more at https://github.com/Cumulocity-IoT/cumulocity-community-plugins/issues/32#issuecomment-2473690942
should be discussed I guess